### DISCUSSION TITLE
how to retrieve invoices that cannot be modified anymore ?

### DISCUSSION THREAD

**Discussion by crismas71** [fattureincloud/api#477] [Q&A] [answered]
ciao, vorremmo intercettare solo le fatture emesse che non subiranno più alcuna modifica in termini di cancellazione della fattura stessa oppure di importo e altri dati. Ho visto che ad esempio le fatture che presentano status ei_status= Sent sono state inviate a SDI e quindi non possono essere più modificate se non con note di credito/debito. 

Ci sono altri stati che configurano questa impossibilità successiva di apportare modifiche oppure cancellarla riconoscibile dalla risposta della get /c/{company_id}/issued_documents ? 

![image](https://github.com/user-attachments/assets/ec430798-192d-49c2-a986-9bd268c355cb)

E invece per quanto riguarda le fatture ricevute (spese) queste possono essere sempre cancellate/modificate oppure ci sono delle limitazioni ? Ad esempio una fattura non ancora registrata (ma che cmq arriva da SDI) NON può cambiare importo ma può essere modificata in qualche altra informazione ? Ho visto che può essere invece cancellata.
E se la fattura è stata registrata può sempre essere cancellata ma non può cambiare importo , è così?

**Comment by valmoz** [comment, marked as answer]
Buongiorno @crismas71 

gli stati bloccanti sono:
- attempt
- pending
- sent
- processing
- not_delivered
- no_response
- accepted
- manual_rejected
- manual_accepted

Sulle spese invece l'unico blocco dovrebbe essere legato alla presenza di un pagamento con TS Pay.

Grazie
Mauro
